source/Interfaces/Menu.cpp

```diff
@@ -50,22 +53,46 @@ namespace i_2D {
      * @param window the main window of the graphic interface
      */
     void Menu::HandleMouseMove(sf::RenderWindow &window) {
```

Comment:
This looks like it will work, but there has to be a cleaner and more future-proof way to handle this.